src/Remapping/distributed_remapping.jl

```diff
@@ -990,115 +1017,45 @@ This can be used directly for plotting.
 )
-    return interpolate(field, axes(field); hresolution, zresolution)
+    return interpolate(field, axes(field); target_hcoords, target_zcoords)
 end
 
+# interpolate, positional
 function interpolate(field::Fields.Field, target_hcoords, target_zcoords)
-    remapper = Remapper(axes(field), target_hcoords, target_zcoords)
-    return interpolate(remapper, field)
-end
-
-"""
-    default_target_hcoords(space::Spaces.AbstractSpace; hresolution)
-
-Return an Array with the Geometry.Points to interpolate uniformly the horizontal
-component of the given `space`.
-"""
-function default_target_hcoords(space::Spaces.AbstractSpace; hresolution = 180)
-    return default_target_hcoords(Spaces.horizontal_space(space); hresolution)
+    return interpolate(field, axes(field); target_hcoords, target_zcoords)
 end
 
-"""
-    default_target_hcoords_as_vectors(space::Spaces.AbstractSpace; hresolution)
-
-Return an Vectors with the coordinate to interpolate uniformly the horizontal
-component of the given `space`.
-"""
-function default_target_hcoords_as_vectors(
+function interpolate(
+    field::Fields.Field,
     space::Spaces.AbstractSpace;
-    hresolution = 180,
-)
-    return default_target_hcoords_as_vectors(
-        Spaces.horizontal_space(space);
-        hresolution,
-    )
-end
-
-function default_target_hcoords(
-    space::Spaces.SpectralElementSpace2D;
-    hresolution = 180,
-)
-    topology = Spaces.topology(space)
-    domain = Meshes.domain(topology.mesh)
-    xrange, yrange = default_target_hcoords_as_vectors(space; hresolution)
-    PointType =
-        domain isa Domains.SphereDomain ? Geometry.LatLongPoint :
-        Topologies.coordinate_type(topology)
-    return [PointType(x, y) for x in xrange, y in yrange]
-end
-
-function default_target_hcoords_as_vectors(
-    space::Spaces.SpectralElementSpace2D;
-    hresolution = 180,
+    target_hcoords,
+    target_zcoords,
 )
-    FT = Spaces.undertype(space)
-    topology = Spaces.topology(space)
-    domain = Meshes.domain(topology.mesh)
-    if domain isa Domains.SphereDomain
-        return FT.(range(-180.0, 180.0, hresolution)),
-        FT.(range(-90.0, 90.0, hresolution))
-    else
-        x1min = Geometry.component(domain.interval1.coord_min, 1)
-        x2min = Geometry.component(domain.interval2.coord_min, 1)
-        x1max = Geometry.component(domain.interval1.coord_max, 1)
-        x2max = Geometry.component(domain.interval2.coord_max, 1)
-        return FT.(range(x1min, x1max, hresolution)),
-        FT.(range(x2min, x2max, hresolution))
-    end
+    remapper = Remapper(space; target_hcoords, target_zcoords)
+    return interpolate(remapper, field)
 end
 
-function default_target_hcoords(
-    space::Spaces.SpectralElementSpace1D;
-    hresolution = 180,
+function interpolate(
+    field::Fields.Field,
+    space::Spaces.FiniteDifferenceSpace;
+    target_zcoords,
+    kwargs..., # needed to capture the unused target_hcoords
```

Comment:
Should we explicitly capture `target_hcoords`? I think that, this way, mis-spelled kwargs are caught early

Reply:
(same elsewhere)

Reply:
Good idea